<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steganografia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>558</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="lImageSource">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QPushButton" name="bEncode">
            <property name="text">
             <string>Encode -&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bDecode">
            <property name="text">
             <string>&lt;- Decode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lImageOutput">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,4,2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item row="4" column="0">
           <widget class="QPushButton" name="bSaveFileDecoded">
            <property name="text">
             <string>Save Decoded Image</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="bSaveMessageDecoded">
            <property name="text">
             <string>Save Decoded Message</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="bLoadMessageEncode">
            <property name="text">
             <string>Load Message to Encode</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="bLoadFileDecode">
            <property name="text">
             <string>Load Image to Decode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="bLoadFileEncode">
            <property name="minimumSize">
             <size>
              <width>143</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Load Image to Encode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <widget class="QtCharts::QChartView" name="chartview"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Avalible space: 0KiB</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>37</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sRed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="pageStep">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lValueRed">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>37</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sGreen">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="pageStep">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lValueGreen">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>37</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sBlue">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="pageStep">
           <number>8</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lValueBlue">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView&gt;
#include &lt;QtCharts/chartsnamespace.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
